packages/core/src/core.ts

```diff
@@ -150,7 +150,7 @@ export function addPath(inputPath: string): void {
  */
 export function getInput(name: string, options?: InputOptions): string {
   const val: string =
-    process.env[`INPUT_${name.replace(/ /g, '_').toUpperCase()}`] || ''
+    process.env[`INPUT_${name.replace(/[- ]/g, '_').toUpperCase()}`] || ''
```

Comment:
This would break many test setups that define, say, `process.env.["INPUT_WHO-TO-GREET"]`. You need some code to maintain backwards-compatibility, i.e.

```js
process.env[`INPUT_${name.replace(/[- ]/g, '_').toUpperCase()}`] ||
process.env[`INPUT_${name.replace(/ /g, '_').toUpperCase()}`] || ''
```

Having said that, I have yet to see any `actions/` PR  getting accepted when the PR author doesn't happen to work at GitHub. I don't understand the policy (at least I assume that there _is_ an internal, not publicly communicated, policy not to accept "outside PRs"...) but that's what I have observed in the past. Not trying to spoil the party here, but wanting to dampen both your and my hopes to reflect realistic expectations.